go/cli/cmd/kagent/main.go

```diff
@@ -209,7 +210,166 @@ func main() {
 
 	getCmd.AddCommand(getSessionCmd, getAgentCmd, getToolCmd)
 
-	rootCmd.AddCommand(installCmd, uninstallCmd, invokeCmd, bugReportCmd, versionCmd, dashboardCmd, getCmd)
+	initCmd := &cobra.Command{
+		Use:   "init [framework] [language] [agent-name]",
+		Short: "Initialize a new agent project",
+		Long: `Initialize a new agent project using the specified framework and language.
+
+You can customize the root agent instructions using the --instruction-file flag.
+You can select a specific model using --model-provider and --model-name flags.
+If no custom instruction file is provided, a default dice-rolling instruction will be used.
+If no model is specified, the agent will need to be configured later.
+
+Examples:
+  kagent init adk python dice
+  kagent init adk python dice --instruction-file instructions.md
+  kagent init adk python dice --model-provider Gemini --model-name gemini-2.0-flash`,
+		Args: cobra.ExactArgs(3),
+		Run: func(cmd *cobra.Command, args []string) {
+			initCfg := &cli.InitCfg{
+				Config:    cfg,
+				Framework: args[0],
+				Language:  args[1],
+				AgentName: args[2],
```

Comment:
This can panic, you need to make sure args is long enough, or use cobra built in validators

Reply:
using `cobra.ExactArgs(3)` to ensure that the required number of args are passed to the cmd here and for the other cmds as well.